README.md

```diff
@@ -56,6 +58,46 @@ JUDOSCALE = {
 
 Once deployed, you will see your request queue time metrics available in the Judoscale UI.
 
+# Using Judoscale with Flask
+
+Import `RequestQueueTimeMiddleware` and wrap `wsgi_app`:
+
+```python
+# app.py
+from judoscale.flask.middleware import RequestQueueTimeMiddleware
+
+app = Flask("MyFlaskApp")
+app.wsgi_app = RequestQueueTimeMiddleware(app.wsgi_app)
+```
+
+This sets up the Judoscale middleware to capture request queue times.
```

Comment:
☝️ I added the bit above based on what I saw in the sample app. Does this correct? Anything you'd change?